saleor/order/models.py

```diff
@@ -360,7 +360,9 @@ class OrderLine(models.Model):
     )
     # max_length is as produced by ProductVariant's display_product method
```

Comment:
Is this comment obsolete?

Reply:
I think yes. This comment explains why `product_name` have `max_length=386`. We can't change max length because we can't get new `product_name` for order lines with removed products or variants.